src/tests/system/tests/test_ipa.py

```diff
@@ -122,3 +124,70 @@ def test_ipa__hostpublickeys_by_ip(client: Client, ipa: IPA, public_keys: list[s
     assert len(public_keys) == len(result.stdout_lines), "Did not get expected number of public keys!"
     for key in public_keys:
         assert f"{ip} {key}" in result.stdout_lines, "Did not get expected public keys!"
+
+
+@pytest.mark.ticket(bz=1926622)
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopology.IPA)
+def test_ipa__check_gssapi_authentication_indicator(client: Client, ipa: IPA):
+    """
+    :title: Verify GSSAPI authentication indicator
+    :description:
+        Checks that the kerberos ticket was pre-authentication using GSSAPI using authentication indicators
```

Comment:
The sentence is strange. 

"Checks that logs for the authentication indicator showing that the kerberos ticket was obtained using GSSAPI"